app/components/header_component.rb

```diff
@@ -42,6 +42,10 @@ def call
   end
 
   class NavigationItem < GovukComponent::HeaderComponent::NavigationItem
+    def active_class
+      ['dfe-header__navigation-item--active'] if active?
```

Comment:
I'm not sure of the reason for many of your recent changes @martikat but do check you aren't now redefining methods needlessly. `brand` wasn't being used but can it? The parent method is below:

https://github.com/x-govuk/govuk-components/blob/main/app/components/govuk_component/header_component.rb
```ruby
    def active_class
      ["#{brand}-header__navigation-item--active"] if active?
    end
```

Reply:
When using brand it defaults to using `govuk-header__...` for the navigation items

Reply:
The initialiser was telling it `dfe` before it got removed